src/actor/monster/monster-sheet.js

```diff
@@ -0,0 +1,318 @@
+/* eslint-disable no-unused-vars */
+import { FabulaUltimaActorSheet } from "../sheet.js";
+// import { FabulaUltimaCharacterGenerator } from "@components/character-generator/character-generator.js";
+// import { FBLRoll, FBLRollHandler } from "@components/roll-engine/engine.js";
+// import localizeString from "@utils/localize-string";
+// import { ActorSheetConfig } from "@utils/sheet-config.js";
```

Comment:
NB: I'd love to prune whatever comments aren't necessary (both here and where they were copied from)

Reply:
Done!